wire-ios/Wire-iOS/Sources/UserInterface/Calling/Helper/Array+CallParticipant.swift

```diff
@@ -22,13 +22,19 @@ import WireSyncEngine
 extension Array where Element == CallParticipant {
 
     var hasMoreThanTwoConnectedParticipants: Bool {
-        return filter(\.state.isConnected).count > 2
+        filter(\.state.isConnected).count > 2
     }
 
     mutating func sortByName(selfStreamId: AVSClient?) {
-        self = self.sorted {
-            $0.streamId == selfStreamId ||
-                $0.user.name?.lowercased() < $1.user.name?.lowercased()
+        sort {
```

Comment:
I think it's equal in the result and using `self = self.sorted` just additionally copies arrays around.